lib/driverProviders/direct.ts

```diff
@@ -43,7 +42,7 @@ export class Direct extends DriverProvider {
             'browserName ' + this.config_.capabilities.browserName +
                 ' is not supported with directConnect.');
     }
-    return q.fcall(function() {});
+    return Promise.resolve(function() {});
```

Comment:
Since this function doesn't need await, let's not make it async. Just have it return `Promise.resolve()`. The empty function was needed when using `q.fcall()`, but it's not anymore.

Reply:
I removed this completely.